src/main/java/appeng/me/GridStorage.java

```diff
@@ -37,11 +37,11 @@
 public class GridStorage implements IGridStorage
 {
 
-	final long myID;
-	final NBTTagCompound data;
-	final GridStorageSearch mySearchEntry; // keep myself in the list until I'm
+	private final long myID;
+	private final NBTTagCompound data;
+	private final GridStorageSearch mySearchEntry; // keep myself in the list until I'm
 	private final WeakHashMap<GridStorage, Boolean> divided = new WeakHashMap<GridStorage, Boolean>();
-	public boolean isDirty = false;
+	private boolean isDirty = false;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "isDirty" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)